frontend/src/pages/Seller/Auth/CreateAccount.tsx

```diff
@@ -1,9 +1,69 @@
 import * as React from "react"
+import styles from "./SellerRegister.module.css";
+import { Input } from '@chakra-ui/react';
+import { Button, ButtonGroup } from '@chakra-ui/react'
+import {useState} from "react";
+import { Link } from '@chakra-ui/react'
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+// @ts-ignore
 const CreateAccount: React.FC<{}> = (props) => {
+   const [fullName, updatedFullName] = useState('');
```

Comment:
By React convention, the setter function of `useState` should be of form `set____`. For this variable, it would be `setFullName`. Apply this to the other `useState` hooks as well.